content/4_Gold/MST.problems.json

```diff
@@ -36,23 +31,37 @@
       "difficulty": "Easy",
       "isStarred": false,
       "tags": ["MST"],
-      "solutionMetadata": {
-        "kind": "USACO",
-        "usacoId": "623"
-      }
+      "solutionMetadata": { "kind": "USACO", "usacoId": "623" }
+    },
+    {
+      "uniqueId": "ks-checksum",
+      "name": "Checksum",
+      "url": "https://codingcompetitions.withgoogle.com/kickstart/round/0000000000436140/000000000068c2c3",
+      "source": "KS",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["MST"],
+      "solutionMetadata": { "kind": "internal" }
+    },
+    {
+      "uniqueId": "cf-472D",
+      "name": "Design Tutorial: Inverse the Problem",
+      "url": "https://codeforces.com/contest/472/problem/D",
+      "source": "CF",
+      "difficulty": "Easy",
+      "isStarred": false,
+      "tags": ["MST"],
+      "solutionMetadata": { "kind": "autogen-label-from-site", "site": "CF" }
     },
     {
       "uniqueId": "cf-1513D",
       "name": "GCD and MST",
       "url": "https://codeforces.com/problemset/problem/1513/D",
       "source": "CF",
-      "difficulty": "Easy",
+      "difficulty": "Normal",
```

Comment:
wait why was this changed

Reply:
I copy pasted it. Apparently it is also normal on deployment.
![image](https://user-images.githubusercontent.com/43394822/119396038-f4528900-bc99-11eb-8c23-6d622f8afbf2.png)